scripts/ci-footprint-and-performance/1-setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -xeuo pipefail
+export PS4='\n+ $(date "+%T.%N") ${BASH_SOURCE}:$LINENO \011'
```

Comment:
Do we need to export this?

Reply:
Yes, I like to have filenames and lines in logs